### PR TITLE
Feat/leverage claim rewards

### PR DESCRIPTION
**Pull request by cryptokage1996**

### CODE REVIEW

File: bindings/src/querier.rs [hunk 1]
```diff
@@ -885,22 +896,67 @@ impl<'a> ElysQuerier<'a> {
...
+        let raw_resp: LeveragelpPositionsResponseRaw =
+            self.leveragelp_query_positions_for_address(address.to_string(), prev_pagination)?;
+        let leverage_reward_data =
+            self.query_leverage_lp_rewards(address.to_string(), raw_resp.get_pools())?;
+
+        let mut usdc = Decimal::zero();
+        let mut eden = Uint128::zero();
+
+        for i in leverage_reward_data.total_rewards {
+            if i.denom == "usdc".to_string() {
```
Comment (amityadav0) [inline bindings/src/querier.rs:927]:
```suggestion
            if i.denom == "uusdc".to_string() {
```

Comment (amityadav0) [reply bindings/src/querier.rs:927]: this should be uusdc, right ?

File: bindings/src/querier.rs [hunk 2]
```diff
@@ -885,22 +896,67 @@ impl<'a> ElysQuerier<'a> {
...
+                usdc = CoinValue::from_coin(&i, self).ok().unwrap().amount_usd;
```
Comment (amityadav0) [inline bindings/src/querier.rs:928]: Need to handle error here

**Review by amityadav0 — COMMENTED** [review]
Please test it on testnet data and add screenshots here. 

Also about fetching all the data in one query, https://github.com/elys-network/elys/blob/3eec4615b43750061bbc4f38df45a3bbf765dd58/x/tier/keeper/query_portfolio_test.go#L116

Query in order using offset: https://github.com/elys-network/elys/blob/3eec4615b43750061bbc4f38df45a3bbf765dd58/x/tier/keeper/query_portfolio_test.go#L90

But we need to check, for how many values we don't need pagination(fetch total) ?
because if there are too many values it won't fit in a transaction

**Comment by Wgil** [comment]
Will merge this to develop and give it a try on devnet

**Pull request merged**